src/common/clipboard.c

```diff
@@ -154,13 +155,20 @@ void guac_common_clipboard_append(guac_common_clipboard* clipboard, const char*
     if (remaining < length)
         length = remaining;
 
+    char output_data[length];
+    memset(output_data, '\0', length);
```

Comment:
Why?

Reply:
I put it in when I was debugging the variable. Removed